extensions/projection/README.md

```diff
@@ -0,0 +1,139 @@
+# Projection Extension Specification (`proj`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+This document explains the fields of the STAC Projection (`proj`) Extension to a STAC Item. Here `proj` is short
+for "projection", and not a reference to the use of the PROJ format for the CRS attribute.
+
+The field names defined herein should be added as fields in the Item's Properties object. 
+
+When specified on an Item, the values are assumed to apply to all Assets in that Item.  For example, an Item may have several related Assets each representing a band for the Item, all of which are in the same native CRS, e.g., a UTM Zone.  However, there may also be assets intended for display, like a preview image or thumbnail, that have been reprojected to a different CRS, e.g., Web Mercator, to better accommodate that use case.  This case of differing projections per Asset is not currently handled by this extension.
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| proj:epsg        | integer\|null   | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:crs         | string \|null   | Recommended. PROJ string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
+| proj:geometry    | Polygon Object  | Recommended. Defines the footprint of this item. |
+| proj:bbox        | [number]        | Recommended. Bounding box of the item in the native CRS |
+| proj:centroid    | Centroid Object | Recommended. Coordinates representing the centroid of the item in the native CRS |
```

Comment:
Wouldn't it make the spec easier to not define a new Centroid object, but just make it a two element array? Would be more inline with the bbox.

Reply:
Also, it feels odd that all fields are recommended. There should be an indication on why the fields are recommended.

Reply:
Centroid is this way so that it can be used directly by visualization tools like Kibana w/ Elasticsearch.  

Removed recommended, agree about that.